cypress/integration/unit_tests/applications.spec.ts

```diff
@@ -34,4 +34,8 @@ describe('Applications testing', () => {
   it('Push a 5 instances application with mixed options into default namespace and check it', () => {
     cy.runApplicationsTest('allTests');
   });
+
+  it('Download manifest from ui and push an app from it',  () => {
+    cy.runApplicationsTest('downloadManifestAndPushApp');
+  });
```

Comment:
Thanks. Added

Reply:
No you did a mistake, you added it to `install_with_default_options.spec.ts` instead of `with_default_options.spec.ts`.
Moreover, I removed the file `install_with_default_options.spec.ts` because we don´t need it anymore.

Reply:
done.